### PR TITLE
[WIP] Skeleton of Memorization Exercise screen #143

### PR DESCRIPTION
**Pull request by alouanemed**
Here is a first look

image

### CODE REVIEW

File: app/src/main/java/com/kanshu/kanshu/Model/FlashcardExerciseOption.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package com.kanshu.kanshu.model;
+
+/**
+ * Created by alouanemed on 17-02-2015.
+ */
+public class FlashcardExerciseOption {
+    private static int nb = 0;
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/Model/FlashcardExerciseOption.java]: What is "nb"? Can you give it a more descriptive variable name?

**Comment by intrications** [comment]
Nice work so far! Shouldn't be too much more before merging.

File: app/src/main/java/com/kanshu/kanshu/FlashCardExerciseActivity.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        return super.onOptionsItemSelected(item);
+    }
+
+    @Override
+    public User getCurrentUser() {
+        return mCurrentUser;
+    }
+
+
+    public class FlashCardPagerAdapter extends FragmentStatePagerAdapter {
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/FlashCardExerciseActivity.java]: Not sure if you saw the comment about this before. Please remove the tab and set the ActionBar title to "Flashcard Exercise".

Comment (alouanemed) [inline app/src/main/java/com/kanshu/kanshu/FlashCardExerciseActivity.java]: oh yeah , I will change that :)

**Comment by alouanemed** [comment]
https://assembly.com/kanshu/bounties/136

**Comment by fernandobuzi** [comment]
@intrications This pull request has been closed by @alouanemed. Let me know if something can be salvaged. Otherwise I'll just release the same bounty again.

@alouanemed I've tipped you 2000 coins for the work you'd already done on this bounty. Like I mentioned, we're trying not to be unfair with you. Feel free to work with us at any moment, and (like I'd said) to request for a revision of a bounty award if you think it is low. ;)

**Pull request closed without merging**